src/instruments/src/PFD/FMA.tsx

```diff
@@ -89,6 +89,7 @@ const A1A2Cell = () => {
     const AThrMode = getSimVar('L:A32NX_AUTOTHRUST_MODE', 'enum');
 
     let text: string | undefined;
+    let autobreakText: string | undefined;
```

Comment:
```suggestion
    let autobrakeText: string | undefined;
```
Auto Break != Auto Brake 😉

Reply:
Yes, sorry for the typo, do you change it or do you want me to change it?